### PR TITLE
RUN-458 Add Rust timers tutorial

### PR DESCRIPTION
**Pull request by dfinity-berestovskyy**

### CODE REVIEW

File: docs/developer-docs/backend/rust/timers.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+* `#[ic_cdk_macros::init]` &mdash; marks the following `init` function as a canister initialization method.
+* `fn init(interval: u64) {...}` &mdash; defines the initialization method. Just like in the `.did` definition, the function takes one argument: timer interval in seconds.
+* `ic_cdk::println!(...)` &mdash; prints the debug log message on the local `dfx` console.
+* `ic_cdk_timers::set_timer_interval(...)` &mdash; creates a new periodic timer with the specified interval and a closure to call.
+* `COUNTER.fetch_add(1, ...)` &mdash; every time the periodic task is triggered, the global `COUNTER` is increased.
+
+Step 6. Implementing canister upgrade
+-------------------------------------
+
+Note, as described in [Periodic Tasks and Timers](/developer-docs/backend/periodic-tasks.md), the timers library does not handle canister upgrades. It is up to the canister developer to serialize the timers in the `canister_pre_upgrade` and reactivate the timers in the `canister_post_upgrade` method if needed.
```
Comment (sesi200) [inline docs/developer-docs/backend/rust/timers.md:138]: link is broken. Preferred way to write this is `../periodic-tasks.md`

File: docs/developer-docs/backend/rust/timers.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+   ```sh
+   % dfx canister call my_timers_backend counter
+   (8 : nat64)
+   ```
+
+More to Explore
+---------------
+
+1. Learn more about periodic tasks and timers in the [Internet Computer Developer Guide.](/developer-docs/backend/periodic-tasks.md)
```
Comment (sesi200) [inline docs/developer-docs/backend/rust/timers.md:228]: also broken

**Review by sesi200 — COMMENTED** [review]
broken links, otherwise looks good

File: docs/developer-docs/backend/rust/timers.md (hunk 1 above)
Comment (dfinity-berestovskyy) [reply docs/developer-docs/backend/rust/timers.md:138]: There are 70 links `(/developer-docs/...`, shall we fix them all (in a separate PR)?

**Review by sesi200 — APPROVED** [review]

**Pull request merged**